src/tag_iterator_util.rs

```diff
@@ -0,0 +1,64 @@
+use ebml_iterable_specification::{EbmlSpecification, EbmlTag};
```

Comment:
I like how you separated these items into a util file - it definitely helps organize things!